### ISSUE TITLE
 Anti-nucleoporin 62 antibody positivity (anti-p62)

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6594]
**Preferred term label:** Anti-nucleoporin 62 antibody positivity (anti-p62)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anti-nucleoporin 62 antibody is an antibody directed against components of the nuclear membrane and is found in primary biliary cirrhosis (PBC).

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
OMIM:109720 Biliary Cirrhosis, Primary

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
